gradle/libs.versions.toml

```diff
@@ -1,17 +1,17 @@
 [versions]
-libraries-checker = "3.22.2"
-libraries-ideProbe = "0.36.1"
+libraries-checker = "3.23.0"
+libraries-ideProbe = "0.40.0"
```

Comment:
Let's pin on 0.37.0 for a moment, coz 0.38.0 downgraded its robot-server dependency accidentally (?) on https://github.com/VirtusLab/ide-probe/pull/198/files#diff-ad2642dc77b3679e4ad2c7d3b2f59a2dcf48c1bf5b93a558a4fa44828315a34c... we'll have a lot of spam in logs with 0.38+

Reply:
```suggestion
# @pin until ide-probe bumps its robot-server dependency back to 0.11.15
libraries-ideProbe = "0.37.0"
```